collector/cpu_linux.go

```diff
@@ -83,8 +114,64 @@ func NewStatCollector() (Collector, error) {
 	}, nil
 }
 
+func (c *cpuCollector) Update(ch chan<- prometheus.Metric) error {
+	err1 := c.UpdateStat(ch)
+	err2 := c.UpdateCpufreq(ch)
+	if err1 != nil {
+		return err1
+	}
+	if err2 != nil {
+		return err2
+	}
+	return nil
+}
+
+// Expose cpu frequency statistics.
```

Comment:
Go's docstring convention says it should be a `comment is a complete sentence that begins with the name of the element it describes.` https://blog.golang.org/godoc-documenting-go-code.